docs(faq): windows on elastic metal MTA-1805 (#446)
scaleway/docs-content: faq/elastic-metal.mdx

## How many flexible IPs can I have per Elastic Metal server?

You can have up to 64 [flexible IPs](/compute/elastic-metal/how-to/order-flexible-ip/) per Elastic Metal server.

## Is Windows available on Elastic Metal?

Windows is available with a monthly license for Elastic Metal servers by the hour and by the month, in French only. You have to order a server from the list of Elastic Metal servers compatible with Microsoft Windows Server. Then you can choose the version of Windows Server to install in one click from our console.

## Why are there different versions of Microsoft Windows Server?

> The Windows Server licenses exist in 2 versions (Standard and Datacenter) with different prices. Depending on the version you choose, you have certain features (roles) available as well as limitations in the Standard version. You can find all the limitations on the Microsoft website.

Reviewer: ```suggestion
The Windows Server license comes in 2 versions (Standard and Datacenter) with different prices. The availability of certain features (such as roles) will be dependent on the version you choose. You can find more information about the various features and limitations of each version on the Microsoft website.
```

## What is the price of Windows Server licenses?

> Windows Server 2019 or 2022 licenses are priced on a monthly basis (no prorating, calendar month) and defined according to 3 criteria: the number of physical CPUs, the number of cores per CPU and the OS version: Standard or Datacenter. You pay your license month by month, with your Elastic Metal whether it is billed per hour or per month.

Reviewer: ```suggestion
Windows Server 2019 or 2022 licenses are priced on a monthly basis (no prorating, based on the calendar month) and are priced according to 3 criteria: the number of physical CPUs, the number of cores per CPU and the OS version: Standard or Datacenter. The license is billed each month, whether your Elastic Metal server itself is billed per hour or per month.
```

## Can I transfer the monthly license to another server ?

> The proposed licenses are only valid on the Elastic Metal servers that we propose and it will not be possible to recover it for an external use.  

Reviewer: ```suggestion
The proposed licenses are only valid on Scaleway's Elastic Metal servers, and cannot be recovered for external use.  
```

> Each installation of Windows imply the invoicing of a new Windows license (except in case of reinstallation of Windows on an Elastic Metal with a Windows installation provided by our installation system).

Reviewer: ```suggestion
Each installation of Windows implies the invoicing of a new Windows license, except where Windows is being reinstalled on an Elastic Metal server where it was already previously installed.
```

> ## How to install Hyper-V on Windows Server?

Reviewer: ```suggestion
## How can I install Hyper-V on Windows Server?
```
